### ISSUE TITLE
Add visual indicator if a template subject does not include the ticket number

### ISSUE THREAD

**Issue by LordMike** [osTicket/osTicket#2171]
Hi,

I had an issue where replies by email were not appended to a ticket, but rather created a new. When looking at #342, I realized I missed the ticket number in the subject (I instead used ticket id - and I didn't enclose it in brackets).

I can understand from #342 that it has been changed to require only #-number in the subject - but that also some other measures are used to detect replies?

In any case, I changed to include the number in the subject, but it would be great if there had been a warning sign next to the subject, explaining that it's best to include "#%{ticket.number}" in the subject - somewhere. So that reply-parsing always works :)

Cheers.

**Comment by greezybacon** [comment]
@LordMike this ticket number is *not* required in the subject line for the system to work properly. With the most recent version, the subject like edit you made is only required for broken mail clients which do not construct responses with the proper `References` and `In-Reply-To` MIME headers

**Comment by LordMike** [comment]
Is there a list of broken clients?
I replied using Gmail, to some webhosting mail server, and then it was picked up via imap or pop3.

After I re-added the number to the subject, it worked again. Hence my suggestion.

-----Original Message-----

**Comment by greezybacon** [comment]
@LordMike what version of osTicket are you using?

**Comment by LordMike** [comment]
An update is that I replied to an email last night, which got added as a completely new ticket - so the issue isn't quite gone yet.

As for version - I have yet to update to the latest, but I'm running `1.8-git`.

**Comment by LordMike** [comment]
Upgraded to `v1.9.9` and the issue I was seeing is not present. To test, I replied to a ticket I created using osticket - I created it directly on the website to avoid triggering mails to other people.

Replying to it via. Gmail appended my response to the ticket, as expected.

Addendum.
My email provider seems to strip some of the details you're referring to. Specifically, of the two mail headers you mentioned, I can in gmail only see one (and it's empty): `References: `.

So for my mailprovider, all I have to go on is the subject, I guess.
